polls/templates/polls/index.html

```diff
@@ -29,8 +29,14 @@ <h6><a href="{% url 'polls:detail' question.id %}">{{ question.question_text }}<
                 <p>No polls are available.</p>
             {% endif %}
         </div>
+      
+        <div class="container">
+            <!-- Link to logout page in loginPage folder -->
+            <a href="http://127.0.0.1:8000/logoutPage/" id="logout">logout</a>
```

Comment:
While it may not matter much for the tests, in the production website these should definitely not be hardcoded to this degree.